animate/adapt.py

```diff
@@ -173,13 +176,12 @@ def adapt(mesh, *metrics, name=None, serialise=False, remove_checkpoints=True):
     """
     nprocs = COMM_WORLD.size
 
-    # Parallel adaptation is currently only supported in 3D
-    if mesh.topological_dimension() != 3:
-        serialise = nprocs > 1
-
-    # If already running in serial then no need to use checkpointing
-    if nprocs == 1:
-        serialise = False
+    PETSc.Sys.Print(f"{serialise=}", comm=COMM_SELF)
```

Comment:
Wow, I've never seen this notation before!

Reply:
Very useful for debugging! That's why I forgot to remove it here, but did in a later commit :)